zanata-war/src/main/java/org/zanata/util/EssentialDataCreator.java

```diff
@@ -101,11 +101,19 @@ public void prepare() {
                 log.info("Creating 'admin' role");
                 if (accountRoleDAO.create("admin", MANUAL, "user",
                         "glossary-admin") == null) {
-                    throw new RuntimeException("Couldn't create 'admin' role");
+                    throw new RuntimeException("Could not create 'admin' role");
                 }
                 adminExists = false;
             }
 
+            if (!accountRoleDAO.roleExists("project-creator")) {
+                log.info("Creating 'project-creator' role");
+                if (accountRoleDAO.create("project-creator", MANUAL, "user") == null) {
+                    throw new RuntimeException(
+                        "Could not create 'project-creator' role");
+                }
+            }
+
```

Comment:
I think we are at (or past) the point where this bit of logic should be extracted into a method, called once for each role we want.  (Or a for loop.)